common/latch_test.cpp

```diff
@@ -87,7 +87,7 @@ TEST(LatchTest, MultiThreaded) {
     called = true;
   });
 
-  std::deque<std::jthread> threads;
+  std::deque<std::thread> threads;
```

Comment:
```suggestion
  std::vector<std::thread> threads;
```
Is there a specific reason for deque? It just kind of says "look at me" since it's not the default choice.

Reply:
For some reason I thought I saw an error with `std::vector` and assumed it was because `std::thread` wasn't movable so switched to `std::deque`. Anyways, `std::vector` totally works fine here, done (plus the clang-tidy fix).